apps/src/lib/levelbuilder/rubrics/EvidenceDescriptionsRow.jsx

```diff
@@ -1,16 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export default function EvidenceDescriptionsRow({isAiEnabled, evidenceLabel}) {
+export default function EvidenceDescriptionsRow({
+  isAiEnabled,
```

Comment:
I was able to take this out in the parent component, but here I believe this needs to be passed in separately because being iaEnabled is stored at the `learning_goal` level of data, not the `learning_goal_evidence_level` level.